java/test/jmri/jmrit/operations/trains/TrainIconTest.java

```diff
@@ -25,6 +26,7 @@ public void testCTor() {
 
     // test TrainIcon attributes
     @Test
+    @Ignore("Initializing PanelEditor takes many seconds, test coverage not worth the wait")
```

Comment:
Instead of ignoring the tests that require a panel editor, we should make use of the beforeclass and afterclass annotations to create/teardown the panels once for the whole test class instead of for each test.

There are some examples of doing this in the layoutEditor tests.  I'm not sure we've done it for any of the PanelEditor tests yet.